cpp/include/cudf/functions.h

```diff
@@ -55,6 +55,18 @@ gdf_error gdf_nvtx_range_pop();
 gdf_error gdf_count_nonzero_mask(gdf_valid_type const *masks,
                                  gdf_size_type num_rows, gdf_size_type *count);
 
+/**
+ * Calculates the number of bytes to allocate for a validity indicator
```

Comment:
I think you should explain what a "validity indicator pseudo-column" is. This is a new bit of vocab for cudf...